models/user.go

```diff
@@ -1137,3 +1137,15 @@ func UnfollowUser(userID, followID int64) (err error) {
 	}
 	return sess.Commit()
 }
+
+// Get the repos starred by a particular user
+func GetStarredRepos(userID int64) ([]*Repository, error) {
+	sess := x.Where("star.uid=?", userID)
+	sess = sess.Join("LEFT", "star", `"repository".id=star.repo_id`)
+	repos := make([]*Repository, 0, 10)
+	err := sess.Find(&repos)
+	if err != nil {
```

Comment:
I think it could be simpler
```Go
repos := make([]*Repository, 0, 10)
err := x.Where("star.uid=?", userID).
        Join("LEFT", "star", `"repository".id=star.repo_id`).
        Find(&repos)
```